### DISCUSSION TITLE
How to show number label at the every single Barchart?

### DISCUSSION THREAD

**Discussion by iam-Justin** [cetz-package/cetz#560] [Q&A]
Like this, how to show it on Barchart and Clustered Barchart?
![Screenshot qwerty](https://github.com/cetz-package/cetz/assets/20792346/d63bc200-51b9-44f5-8064-cc1e4fcde8e3)
I read the manual but still can't figure out if it works and how to do it.
Thank You!~

**Comment by johannes-wolf** [comment]
There is a stale PR for it. As a workaround, you can add one `content` call inside a `plot.annotate(...)` body for rendering the labels at the correct position. Plot scaling might have to be adjusted by hand though.
